server/src/http/refresher/feature_refresher.rs

```diff
@@ -403,15 +403,24 @@ impl FeatureRefresher {
             .unleash_client
             .get_client_features_delta(ClientFeaturesRequest {
                 api_key: refresh.token.token.clone(),
-                etag: refresh.etag.clone(),
+                etag: None,
```

Comment:
No point to pass in this etag, because etags are different for old api and delta api.

Reply:
I'm curious, don't we still need an etag here?